i3-mousedrag is from git source rather than a proper release distfile so
normally has debugging on by default as per i3 build config; disable as
debug logging has a lot of %s null which are very noisy in syslog

Index: libi3/is_debug_build.c
--- libi3/is_debug_build.c.orig
+++ libi3/is_debug_build.c
@@ -15,6 +15,8 @@
  *
  */
 bool is_debug_build(void) {
+    return false;
+
     /* i3_version contains either something like this:
      *     "4.0.2 (2011-11-11)" (release version)
      * or: "4.0.2-123-gC0FFEE"  (debug version)
